drivers/soundwire/intel.c

```diff
@@ -1579,55 +1567,52 @@ static int intel_master_startup(struct sdw_master_device *md)
 	 * definition of Master properties.
 	 */
 	if (!(link_flags & SDW_INTEL_MASTER_DISABLE_PM_RUNTIME_IDLE))
-		pm_runtime_idle(&md->dev);
+		pm_runtime_idle(sdw->cdns.dev);
 
 	return 0;
 
 err_interrupt:
 	sdw_cdns_enable_interrupt(&sdw->cdns, false);
 err_init:
-	sdw_bus_master_delete(&sdw->cdns.bus);
 	return ret;
 }
 
-static int intel_master_remove(struct sdw_master_device *md)
+static int intel_master_remove(struct platform_device *pdev)
 {
-	struct sdw_intel *sdw;
-
-	pm_runtime_disable(&md->dev);
```

Comment:
is this on purpose? Is it because it's now done in `sdw_intel_cleanup()`

Reply:
> is this on purpose? Is it because it's now done in `sdw_intel_cleanup()`

Yes, and it is done in sdw_master_device_del() now.